backend/main.go

```diff
@@ -263,5 +263,5 @@ func categorizeItems(orderHistory *[]OrderInfo) ItemCategories {
 }
 
 func main() {
-	lambda.Start(Handler)

```

Comment:
does this line ever need to be used when deploying to lambda? if yes, leave it commented out w/ a comment saying why

if it's completely unneeded, then thank you for removing it :)